Formula/k/klee.rb

```diff
@@ -40,6 +43,11 @@ class Klee < Formula
     sha256 "8b3cfd7bc695bd6cea0f37f53f0981f34f87496e79e2529874fd03a2f9dd3a8a"
   end
 
+  resource "tabulate" do
+    url "https://files.pythonhosted.org/packages/ec/fe/802052aecb21e3797b8f7902564ab6ea0d60ff8ca23952079064155d1ae1/tabulate-0.9.0.tar.gz"
+    sha256 "0095b12bf5966de529c0feb1fa08671671b3368eec77d7ef7ab114be2c068b3c"
+  end
```

Comment:
`tabulate` has a CLI tool, so we don't need to get rid of it.

Reply:
I think our decision for CLI tools of PyPI-distributed Python packages are they need to be installed as virtualenvs (falling under terms of https://docs.brew.sh/Python-for-Formula-Authors#installing-applications).

Exposing libraries in `HOMEBREW_PREFIX/lib` would need to meet the new stricter exceptions (which we should audit, e.g. https://github.com/Homebrew/brew/pull/16663)